app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict]) -> list[Person]:
+    list_person = [
+        Person(person.get("name"), person.get("age")) for person in people
+    ]
+    for person in people:
+        if person.get("wife"):
+            wife = Person.people[person["wife"]]
+            Person.people[person["name"]].wife = wife
+        elif person.get("husband"):
+            husband = Person.people[person["husband"]]
+            Person.people[person["name"]].husband = husband
```

Comment:
Person.people.get(person.get("name"))

create a separated variable for it to not duplicate code
use get() method instead of parenthesis everywhere to avoid KeyError